magento/pwa-studio
# Remove unnecessary dependencies in storefront template

## Description

Removes unnecessary dependencies in the storefront template project.

## Related Issue

Closes PWA-2916

## Acceptance





Any core developer

### Verification Stakeholders



Any core developer

### Specification




## Verification Steps





1. Run `yarn watch:venia`
2. Navigate to the application
3. Verify event data is not being logged in the console

#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
